### ISSUE TITLE
ion-virtual-scroll deprecated

### ISSUE THREAD

**Issue by RidClick** [eakoriakin/ionic-select-searchable#359]
Hello, In the latest ionic/cli updates a warning message has appeared.

```
[Deprecation Warning]: ion-virtual-scroll has been deprecated and will be removed in Ionic Framework v7.0.
See https://ionicframework.com/docs/angular/virtual-scroll for migration steps.
```

I just wondering if you will update the ionic-selectable with the ionic recommendation angular/cdk.

Thanks.

**Comment by edy-ap** [comment]
Thanks we will update this component,

**Comment by Scooby27** [comment]
Hello, are there any updates to this? Ionic 7 has now been released and removed ion-virtual-scroll.

**Comment by vorkapps** [comment]
Hi! Any news on the update to this component to be compatible with Ionic 7 now that virtual-scroll has been deprecated? Any ETA? Thanks!

**Comment by NLueg** [comment]
I don't think that the creator of the library is maintaining the library anymore 😞
